book.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
     title: 'ZeroMQ RFCs',
-    gitbook: '3.0.0-pre.15'
+    gitbook: '3.0.0-pre.15',
```

Comment:
Is it guaranteed to pick up the latest beta both on gitbook.com and when using gitbook locally?